docs/timeseries/index.rst

```diff
@@ -252,8 +252,11 @@ Let's take a look at the folded time series:
 Using the :ref:`stats` module, we can normalize the flux by sigma-clipping
 the data to determine the baseline flux::
 
+    >>> import warnings
     >>> from astropy.stats import sigma_clipped_stats
-    >>> mean, median, stddev = sigma_clipped_stats(ts_folded['sap_flux'])  # doctest: +REMOTE_DATA
+    >>> with warnings.catch_warnings():
+    ...     warnings.simplefilter('ignore')  # Ignore warning about invalid values
```

Comment:
I guess we can use the new `IGNORE_WARNINGS` when a new version of `pytest-doctestplus` is out.

Reply:
This can now be done in principle!